<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPanel</class>
 <widget class="QWidget" name="SearchPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="hideButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="replaceButton">
     <property name="text">
      <string>&amp;Replace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="prevButton">
     <property name="text">
      <string>&amp;Previous</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="findLayout">
     <item>
      <widget class="QLineEdit" name="findEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>&amp;Next</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="replaceLayout">
     <item>
      <widget class="QLineEdit" name="replaceEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="replaceAllButton">
     <property name="text">
      <string>Replace &amp;all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>&amp;Find:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>findEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Rep&amp;lace:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>replaceEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <layout class="QHBoxLayout" name="optionsLayout">
     <item>
      <widget class="QCheckBox" name="regularBox">
       <property name="text">
        <string>Regular &amp;expression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseBox">
       <property name="text">
        <string>Case &amp;sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wordsBox">
       <property name="text">
        <string>Match whole &amp;words</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="optionsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>prevButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>regularBox</tabstop>
  <tabstop>caseBox</tabstop>
  <tabstop>wordsBox</tabstop>
  <tabstop>hideButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>hideButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchPanel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
